index.ts

```diff
@@ -98,6 +100,38 @@ async function run() {
       await writeVersion(newVersion);
       break;
     }
+    case "pre": {
+      let subAction = params[0] as "" | "major" | "minor" | "patch" ?? ""
+      let identifier = params[1]
+      if (subAction && !["major", "minor", "patch"].includes(subAction)) {
+        identifier = subAction
```

Comment:
nit: can you run this through `deno fmt` so we get the right semicolons/etc in place? I'll add a "contributing" section to the readme with this request shortly

Reply:
derp, yeah